### DISCUSSION TITLE
Add timer for Hunter meats' delayed heal

### DISCUSSION THREAD

**Discussion by fuzxi** [runelite/runelite#17870] [Ideas]
[Hunter meats](https://oldschool.runescape.wiki/w/Hunter_Guild#Hunter_meats) heal an initial amount of HP like normal food, then an additional amount 8 ticks later. It would be nice to have a timer to see when the delayed heal will be applied.

**Comment by DapperMickie** [comment]
Im working on a hub plugin that does exactly that:
https://github.com/runelite/plugin-hub/pull/6043
Oh btw it's 7 ticks :)

**Reply by fuzxi** [reply]
Looks good! I hope they add that to the built-in plugins, it would be strange to have it separated from timers for overloads, prayer enhance, etc.

>it's 7 ticks

 Oh thanks, I was just going off the wiki.

**Reply by DapperMickie** [reply]
The wiki says 7 ticks right?

**Reply by fuzxi** [reply]
The page I linked in the first post says "Hunter meats, when eaten, will heal an initial amount before doing so again after after 8 ticks (4.8 seconds),"

**Comment by DapperMickie** [comment]
Here's a quick example of how it works currently: https://github.com/DapperMickie/delayed-healing/assets/10809203/2bcc771c-ec26-4c47-b1b1-009618ea00d3
